app/src/main/java/com/example/taskintern/model/Weather.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.taskintern.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
```

Comment:
Deprecate olmus parcelize import edilmis. Bunu class tan import edelim. 
```suggestion
import kotlinx.parcelize.Parcelize
```
Bunu ide'yi acmadan gormek zor, ide zaten uyari gosteriyor. Bu ide uyarilarini dikkate alalim